client/layout/logged-out.jsx

```diff
@@ -134,6 +136,7 @@ export default connect( state => {
 	const currentRoute = getCurrentRoute( state );
 	const isJetpackLogin = currentRoute === '/log-in/jetpack';
 	const noMasterbarForRoute = currentRoute === '/log-in/jetpack';
+	const isPopup = typeof window !== 'undefined' && window.opener && window.opener !== window;
```

Comment:
Ideally, server-rendered markup should be identical to the client-rendered one. React is rather sensitive about that and mail fail to hydrate the SSR-ed page.

Here, the SSR-ed page requested by the popup window will be rendered with `isPopup=false`, and the initial client-side render will be rendered with `isPopup=true`. Not 100% isomorphic.

Adding a `popup=true` query arg would be one of possible solutions.

Reply:
Thanks @jsnajdr - working on that server-side in D33024-code